### DISCUSSION TITLE
We are looking for paid developers to work on V and related projects.

### DISCUSSION THREAD

**Discussion by medvednikov** [vlang/v#10237] [Announcements]
5/10/15/30 hours per week, flexible schedule, fully remote.

Experience with compilers/parsers is welcome.

One of the top priorities right now is the V language server.

If you are interested or know someone who could be interested, please send an email to alexander@vlang.io

**Comment by putao0525** [comment]
哥们加油

**Comment by ttcoder** [comment]
I know I'm late, but...

Are you still looking for (part time) paid help ?

Probably best for someone to ask here before "cold-mailing" you three years after the OP I guess

Thanks !
